### ISSUE TITLE
Send messages with markdown

### ISSUE THREAD

**Issue by cubetastic33** [poljar/weechat-matrix#179]
Is it currently possible to send messages with markdown formatting? Messages sent from other clients that were formatted render properly, but I couldn't figure out how to send formatted messages from here - for example, I tried `_foo_`, `<em>foo</em>`, `<i>foo<i>`, but none of it worked.

**Comment by poljar** [comment]
The standard [Weechat color codes](https://weechat.org/files/doc/devel/weechat_user.en.html#command_line_colors) work, besides that `*foo*` works for emphasis and backticks for inline code.

**Comment by cubetastic33** [comment]
Ohh that's awesome! But right now we can't use blockquotes, strikethrough, and multiline code? Also, do you have a matrix room for weechat-matrix?

**Comment by poljar** [comment]
Yeah, that's correct. We need a better input parser that supports multiple lines. At this point it's unlikely that it'll get added to the Python script but a Rust rewrite of the whole thing is currently going on.

The room can be found in the badge in the readme:

 [![#weechat-matrix](https://img.shields.io/badge/matrix-%23weechat--matrix:termina.org.uk-blue.svg?style=flat-square)](https://matrix.to/#/!twcBhHVdZlQWuuxBhN:termina.org.uk?via=termina.org.uk&via=matrix.org)
